.npmignore

```diff
@@ -0,0 +1,14 @@
+*.log
```

Comment:
Not a critique, but a question: What's the difference between specifying stuff here vs. in `package.json`'s `files` array?

Reply:
I saw an npm boilerplate that used npmignore also so I guess I thought that was standard. The more I read about it, the more it sounds like the files key + gitignore should just take care of that. I'll publish a new version without `npmignore` and double check that it only adds what we want.